core/tinygo/core.go

```diff
@@ -349,27 +349,27 @@ func CDecodeHealthCheckResponse(d *cbor.Decoder) (HealthCheckResponse, error) {
 
 // initialization data for a capability provider
 type HostData struct {
-	HostId             string
-	LatticeRpcPrefix   string
-	LinkName           string
-	LatticeRpcUserJwt  string
-	LatticeRpcUserSeed string
-	LatticeRpcUrl      string
-	ProviderKey        string
-	InvocationSeed     string
-	EnvValues          HostEnvValues
-	InstanceId         string
+	HostId             string        `json:"host_id"`
+	LatticeRpcPrefix   string        `json:"lattice_rpc_prefix"`
+	LinkName           string        `json:"link_name"`
+	LatticeRpcUserJwt  string        `json:"lattice_rpc_user_jwt"`
+	LatticeRpcUserSeed string        `json:"lattice_rpc_user_seed"`
+	LatticeRpcUrl      string        `json:"lattice_rpc_url"`
+	ProviderKey        string        `json:"provider_key"`
+	InvocationSeed     string        `json:"invocation_seed"`
+	EnvValues          HostEnvValues `json:"env_values"`
+	InstanceId         string        `json:"instance_id"`
 	// initial list of links for provider
-	LinkDefinitions ActorLinks
+	LinkDefinitions ActorLinks `json:"link_definitions"`
 	// list of cluster issuers
-	ClusterIssuers ClusterIssuers
+	ClusterIssuers ClusterIssuers `json:"cluster_issuers"`
 	// Optional configuration JSON sent to a given link name of a provider
 	// without an actor context
-	ConfigJson string
+	ConfigJson string `json:"config_json"`
 	// Host-wide default RPC timeout for rpc messages, in milliseconds.  Defaults to 2000.
-	DefaultRpcTimeoutMs uint64
+	DefaultRpcTimeoutMs uint64 `json:"default_rpc_timeout_ms"`
 	// True if structured logging is enabled for the host. Providers should use the same setting as the host.
-	StructuredLogging bool
+	StructuredLogging bool `json:"structured_logging"`
 }
```

Comment:
One of the things I needed to do in the Provider SDK is Unmarshal the host data, read at run time, which needed these json tags.  They are NOT generated...need to figure out how to generate that